src/components/PeopleManagement/LearnerDetailPage/EnrollmentCard.jsx

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  Badge, Card, Hyperlink, Stack,
+} from '@openedx/paragon';
+import { getConfig } from '@edx/frontend-platform';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import { camelCaseDict } from '../../../utils';
+
+const EnrollmentCard = ({ enrollment, enterpriseSlug }) => {
+  const courseEnrollment = camelCaseDict(enrollment);
+  const renderBadge = () => {
+    switch (courseEnrollment.courseRunStatus) {
+      case 'completed': {
+        return (<Badge variant="light">Completed</Badge>);
+      }
+      case 'in_progress': {
+        return (<Badge variant="success">In Progress</Badge>);
+      }
+      default: {
+        return (<Badge variant="info">Upcoming</Badge>);
+      }
+    }
+  };
+
+  return (
+    <Card className="mt-4 p-4">
+      <Stack direction="horizontal" className="justify-content-between">
+        <h3 className="mb-1">{courseEnrollment.displayName}</h3>
+        {renderBadge()}
+      </Stack>
+      <p className="small">{courseEnrollment.orgName} • {courseEnrollment.courseType}</p>
```

Comment:
`courseEnrollment.courseType` appears to be the slug values, while the Figma calls for human readable text like 'Executive Education'.
![image](https://github.com/user-attachments/assets/f723a8f7-6028-4667-8e83-f8677e4e7732)